aikau/src/main/resources/alfresco/forms/controls/NumberSpinner.js

```diff
@@ -55,11 +55,22 @@ define(["alfresco/forms/controls/BaseFormControl",
        *
        * @instance
        * @type {number}
-       * @default null
+       * @default
```

Comment:
I've just checked JSDoc docs and yes, you're right this is much better patten; we don't need to specify the default value - good spot.